### PR TITLE
Adding simple rgb tutorial to the tutorial gallergy

### PR DESCRIPTION
**Pull request by annacasto**
**Describe your changes**
Adding a simple RGB workflow to the tutorial gallery. This tutorial is a straight-forward computer vision workflow and contains references to other workflows with options for more complexity. 

**Type of update**
Is this a:
* New feature or feature enhancement

**Associated issues**
None

**Additional context**

**For the reviewer**
See [this page](https://plantcv.readthedocs.io/en/latest/pr_review_process/) for instructions on how to review the pull request.
- [x] PR functionality reviewed in a Jupyter Notebook
- [x] All tests pass
- [x] Test coverage remains 100%
- [x] Documentation tested
- [x] New documentation pages added to `plantcv/mkdocs.yml`
- [x] Changes to function input/output signatures added to `updating.md`
- [x] Code reviewed
- [ ] PR approved

### CODE REVIEW

**Comment by annacasto** [comment]
Keely, I updated the index.ipynb notebook with the optional color correction step, so it should be all set for final review now.

**Review by k034b363 — COMMENTED** [review]
Looks good to me, just one typo to fix before the histogram (says "Creating a histogram of pixel insenity"). I also might suggest, since this is supposed to be super basic, that it would be helpful to include something like "you must do math to get the height and width of the ROI".

**Comment by annacasto** [comment]
Fixed typos and added some more context

**Review by k034b363 — APPROVED** [review]

**Comment by annacasto** [comment]
Fixed the title of the tutorial and added Workflow Inputs

**Review by HaleySchuhl — CHANGES_REQUESTED** [review]
WorkflowInputs error resolve with re-running from the top, and maybe extra buttons (since binder is inconsistently functional)

File: docs/tutorials/simple_rgb_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Tutorial: Simple RGB workflow tutorial
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/danforthcenter/plantcv-tutorial-simple-rgb-workflow.git/HEAD) Check out our interactive tutorial!
```
[the remark this answers is not in the document]
Comment (annacasto) [reply docs/tutorials/simple_rgb_tutorial.md:3]: Awesome thanks! I wasn't sure what we wanted that to look like. I'll add in those buttons.

[the remark this answers is not in the document]
Comment (annacasto) [reply docs/tutorials/simple_rgb_tutorial.md]: Buttons added!

**Comment by nfahlgren** [comment]
@annacasto could you rerun the notebook and sync it to the tutorial repo? Currently it is rendered with an error:

Screenshot 2024-05-14 at 10 35 22 AM

**Pull request merged**